tdrs-backend/tdpservice/parsers/test/data/fra_empty_first_row.csv

```diff
@@ -0,0 +1,14 @@
+
+202401,946412419
+202401,995040265
+202402,909360961
+202402,987777434
+202403,973351423
+202403,909175891
+202403,909175891
+
+# All data below has errors of some sort
+202301,9000865507
```

Comment:
```suggestion
202301,0000865507
```
this one would capture the all zeros for area number, all zeros for the group number, and 10 digits 😄